.github/workflows/s3-tests.yml

```diff
@@ -0,0 +1,318 @@
+name:  S3 tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - support/**
+    types: [opened, synchronize]
+    paths-ignore:
+      - '**/*.md'
+  release:
+    types:
+      - published
+  workflow_dispatch:
+    inputs:
+      neofs-s3-gw_ref:
+        description: 'neofs-s3-gw ref. Default ref - latest master. Examples: v0.27.0, 8fdcc6d7e798e6511be8806b81894622e72d7fdc, branch_name'
+        required: false
+        default: ''
+
+permissions: write-all
+
+env:
+  S3_TESTS_CONFIG: s3tests.conf
+  S3_GW_IP: 192.168.130.82
+  S3_GW_PORT: 8080
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Set RUN_ID
+        env:
+          TIMESTAMP: ${{ steps.date.outputs.timestamp }}
+        run: echo "RUN_ID=${{ github.run_number }}-$TIMESTAMP" >> $GITHUB_ENV
+
+      - name: Checkout neofs-s3-gw repository
```

Comment:
```
      - name: Checkout neofs-node repository
        uses: actions/checkout@v3
        with:
          path: neofs-s3-gw
```

Reply:
@vvarg229, why `neofs-node` here? We should be using the latest stable in these tests.

Reply:
> @vvarg229, why `neofs-node` here? We should be using the latest stable in these tests.

Yeah, sure, sorry, that's a typo

Reply:
Don't get your point, but I use latest checkout for s3-gw:

`      - name: Checkout neofs-s3-gw repository
        uses: actions/checkout@v3
        with:
          repository: nspcc-dev/neofs-s3-gw
          ref: ${{ github.sha }}
          path: neofs-s3-gw`

Reply:
You may not use those lines in the code:
> repository: nspcc-dev/neofs-s3-gw
>           ref: ${{ github.sha }}

Reply:
Removed it